core/src/main/java/org/ldaptive/handler/JoinEntriesResultHandler.java

```diff
@@ -0,0 +1,39 @@
+package org.ldaptive.handler;
+
+import org.ldaptive.LdapUtils;
+import org.ldaptive.SearchResponse;
+
+import java.util.Arrays;
+import java.util.Objects;
+/**
+ * Joins the values of the attributes in all entries into a single entry.
+ *
+ * @author  Middleware Services
+ */
+public class JoinEntriesResultHandler extends AbstractEntryHandler<SearchResponse> implements SearchResultHandler {
```

Comment:
What do you think about naming this `MergeResultHandler` ?

Reply:
I think it is better. I will change it now, review and push again. Best regards